src/index.js

```diff
@@ -67,36 +81,39 @@ $(document).ready(function() {
       if (mocha) {
         clearInterval(mochaCheck);
         mocha.setup('bdd');
-        const testDiv = document.createElement('div');
-        testDiv.style.position = 'inherit';
-        testDiv.innerHTML = testSuiteSkeleton;
-        document.body.appendChild(testDiv);
-        // Once testDiv is loaded:
+
+        // Once testFrame is loaded:
         let projectTitleCase = localStorage.getItem('projectTitleCase');
         // projectName variable is defined in our example projects so the
         // correct test suite is automatically loaded. This sets default text
         // for <option> text and project indicator in top right corner.
         if (typeof projectName !== 'undefined') {
           projectNameLocal = projectName;
         }
+        // #mocha <div> won't seem to work within iFrame
+        const mochaModal = document.createElement('div');
+        mochaModal.style.position = 'inherit';
+        mochaModal.innerHTML = modalSkeleton;
+        document.body.appendChild(mochaModal);
 
         if ((!projectNameLocal) && (projectTitleCase === null)) {
-          document.getElementById('placeholder').innerHTML = '- - -';
-          document.getElementById(
-            'fcc_test_suite_indicator_wrapper'
+          testFrameBody.querySelector('#placeholder').innerHTML = '- - -';
```

Comment:
Is there a reason to change from `getElementById` to `querySelector`? `getElementById` is supposedly a lot faster. But more importantly, as a developer I can read `getElementById` faster because I don't have to take the extra mental step of figuring out that the CSS selector in question is actually an ID. "Is ID a pound sign or a dot? Oh, class has a dot right? Wait, I better go check that just to make sure...."

Reply:
Switching back to `getElementById` in next update. Thanks!